### DISCUSSION TITLE
mkdocs-material markdown badge

### DISCUSSION THREAD

**Discussion by juftin** [squidfunk/mkdocs-material#6375] [Ideas]
I thought it would be nice to have an official `mkdocs-material` badge for projects to proudly display their docs framework. Here's an example I made:

```markdown
[![mkdocs-material](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/juftin/mkdocs-material/66d65cf/src/templates/assets/images/badge.json)]()
```

[![mkdocs-material](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/juftin/mkdocs-material/66d65cf/src/templates/assets/images/badge.json)]()

https://github.com/squidfunk/mkdocs-material/blob/66d65cf671a7467867c5b184863e796772d3af16/src/templates/assets/images/badge.json#L1-L8

(see [this fork](https://github.com/squidfunk/mkdocs-material/compare/master...juftin:mkdocs-material:master) for an example)

**Comment by squidfunk** [comment]
Thanks for suggesting! Yes, that's definitely a good idea. In fact, we're waiting that shields.io updates its version of SimpleIcons, because [as of last week, Material for MkDocs is part of the SimpleIcons library](https://simpleicons.org/?q=mkdocs), thanks to https://github.com/squidfunk/mkdocs-material/discussions/6304 ☺️ Once this is updated, we add some example badges to our documentation.

**Reply by juftin** [reply]
The logo is out and available! Curious what design you'll go with...

[![mkdocs-material](https://img.shields.io/badge/mkdocs--material-gray?logo=materialformkdocs)](https://github.com/squidfunk/mkdocs-material)

```markdown
[![mkdocs-material](https://img.shields.io/badge/mkdocs--material-gray?logo=materialformkdocs)](https://github.com/squidfunk/mkdocs-material)
```

https://img.shields.io/badge/mkdocs--material-gray?logo=materialformkdocs

**Reply by juftin** [reply]
Oh wait... here it is: https://squidfunk.github.io/mkdocs-material/blog/2023/11/30/adding-a-badge-to-your-project/

[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)

It looks great. You might also want to recommend a non-for-the-badge version too:

[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)

```markdown
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)
```
